rules/go_rules.build_defs

```diff
@@ -913,10 +906,15 @@ def go_get(name:str, get:str|list, repo:str='', deps:list=[], exported_deps:list
             outs += [getroot]
         else:
             for i in installone:
-                if i.startswith(getroot):
-                    outs += [_install_output(getroot, i.removeprefix(getroot))]
+                if not i:
+                    outs += [getroot + ".a"]
+                elif i.endswith("..."):
```

Comment:
`i.endswith("/...")`, perhaps? Although it shouldn't really make any difference...